source/ed/data.h

```diff
@@ -52,10 +53,16 @@ namespace ed {
 template <typename T>
 void normalize_uri(std::vector<T*>& vec) {
     std::string prefix = navitia::type::static_data::get()->captionByType(T::type);
-    for (auto* element : vec) {
-        // Suppression des espaces de l'URI
-        boost::algorithm::replace_all(element->uri, " ", "");
-        element->uri = prefix + ":" + element->uri;
+
+    if (prefix == "stop_point") {
```

Comment:
If you don't want to remove the spaces from the stop_points, why not just remove call the the `normalize_uri` fonction on the stop_points collection ? Instead of adding this test `(prefix == "stop_point")`